api/controllers/location.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+const mongoose = require('mongoose')
+require('../models/Sample') // Register model
+const Sample = mongoose.model('Sample')
+
+const DEFAULT_RSSI = 0
+const K1 = 10
+const K2 = 10
+
+module.exports = {
+  getLocation: async (req, res) => {
+    const buildingId = req.body._id
+    const query = Sample.find({ buildingId: buildingId })
+    query.lean()
+
+    try {
+      const samples = await query.exec()
+
+      if (samples === null) {
+        return res.status(404).json({})
+      }
+
+      // Do algorithm
+      const mainFingerprintSortedSSIDs = sortSSIDs(req.body.fingerprint)
```

Comment:
```suggestion
      const mainFingerprintSortedSSIDs = sortSSIDs(req.body)
```

El JSON que mandamos es el fingerprint directo, no es `{fingerprint: {...}}`
El `name` que se le pone en el swagger [acá](https://github.com/PF-ITBA-SPI/SPI-api/pull/5/files#diff-702be237a87f80c305c9755b62bc42f9R450) es para documentación, no te lo mapea ahí adentro

Reply:
Igual acabo de caer...esto anduvo cuando lo probamos, qué onda? Después lo corro en debug y veo cómo le llega

Reply:
O sea, lo que estaba haciendo era mandar un fingerprint y un id (el del building). Cuando se cambie todo eso va a dejar de ser asi, asi que esta bien la correccion.